recipes/neo4j-driver/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "neo4j-driver" %}
+{% set version = "4.4.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/neo4j-driver-{{ version }}.tar.gz
+  sha256: 955d74f97705966e985639be879f083316185c3978d6908cb8906993b6f6bfca
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - neo4j-python-driver
+    - pip
+  run:
+    - python >=3.6
+    - pytz
+
+test:
+  imports:
+    - neo4j
+    - testkitbackend
```

Comment:
This sounds like testing code is erroneously packaged. Can you delete this and report it upstream?

Reply:
- testkitbackend is not added when I use grayskull - removed from meta.yaml

Reply:
They extra module is still there. BTW, that was fixed in the latest version. If you need this older version you will need to manually delete it like @xhochy requested above.